lib/optimizely/project_config.rb

```diff
@@ -126,16 +126,16 @@ def initialize(datafile, logger, error_handler)
       @all_experiments = @experiment_key_map.merge(@rollout_experiment_id_map)
       @all_experiments.each do |key, exp|
         variations = exp.fetch('variations')
-        @variation_id_map[key] = generate_key_map(variations, 'id')
-        @variation_key_map[key] = generate_key_map(variations, 'key')
-
         variations.each do |variation|
           variation_id = variation['id']
+          variation['featureEnabled'] = variation['featureEnabled'].present?
```

Comment:
Please use 'has_key' instead of 'present'. Present is not available by default in Ruby it's part of the active support extension. See this issue here https://github.com/optimizely/ruby-sdk/pull/63#issuecomment-332894838

Reply:
changed as discussed